joomla/CVE-2023-23752/docker-compose-vulnerable.yml

```diff
@@ -0,0 +1,17 @@
+services:
+  joomla:
+    image: joomla:4.2.6-php8.0
+    restart: always
+    ports:
+      - 8000:80
+    environment:
+      JOOMLA_DB_HOST: joomladb
+      JOOMLA_DB_PASSWORD: example
+    depends_on:
+      - joomlaDb
+
+  joomlaDb:
+    image: mysql:5.6
+    restart: always
```

Comment:
```suggestion
```
Let's remove the `restart` so the container doesn't accidentally restart by itself after the test.